services/src/main/java/org/fao/geonet/api/tools/i18n/TranslationApi.java

```diff
@@ -261,10 +261,11 @@ public void deleteTranslations(
         ServletRequest request,
         HttpServletRequest httpRequest
     ) throws Exception {
-        final ServiceContext context = ApiUtils.createServiceContext(httpRequest);
-        Locale locale = languageUtils.parseAcceptLanguage(request.getLocales());
-        String language = isoLanguagesMapper.iso639_2T_to_iso639_2B(locale.getISO3Language());
-        return translationPackBuilder.getPack(language, pack, context);
+        try (ServiceContext context = ApiUtils.createServiceContext(httpRequest)) {
+            Locale locale = languageUtils.parseAcceptLanguage(request.getLocales());
+            String language = isoLanguagesMapper.iso639_2T_to_iso639_2B(locale.getISO3Language());
+            return translationPackBuilder.getPack(language, pack, context);
+        }
```

Comment:
```suggestion
      try (ServiceContext context = ApiUtils.createServiceContext(httpRequest)) {
        Locale locale = languageUtils.parseAcceptLanguage(request.getLocales());
        String language = isoLanguagesMapper.iso639_2T_to_iso639_2B(locale.getISO3Language());
        return translationPackBuilder.getPack(language, pack, context);
      }
```

Reply:
Outdenting the try / finally statement by two characters can vastly reduce the number of lines in the diff as the origional logic remains unchanged.